providers/implementations/encode_decode/decode_epki2pki.c

```diff
@@ -87,13 +88,9 @@ static int epki2pki_decode(void *vctx, OSSL_CORE_BIO *cin, int selection,
     struct epki2pki_ctx_st *ctx = vctx;
     BUF_MEM *mem = NULL;
     unsigned char *der = NULL;
-    const unsigned char *pder = NULL;
     long der_len = 0;
-    X509_SIG *p8 = NULL;
-    PKCS8_PRIV_KEY_INFO *p8inf = NULL;
-    const X509_ALGOR *alg = NULL;
     BIO *in = ossl_bio_new_from_core_bio(ctx->provctx, cin);
-    int ok = 0;
+    int ok;
```

Comment:
Why?

Reply:
In general, variables that are unconditionally initialised on first use are better not explicitly initialised, then failure to set them unconditionally will turn into compiler warnings. Here, it is admittedly more of a judgement call. If you strongly feel that initialising to 0 is better, I can change it back. But perhaps simplest to let the change happen, either way is fine.

Reply:
I'd better keep it, I strongly prefer the explicit error state